.travis.yml

```diff
@@ -20,7 +20,19 @@ matrix:
 before_install:
   - docker run -d --name=taurus-test -h taurus-test --volume=`pwd`:/taurus $DOCKER_IMG
   - sleep 10
+  - docker exec taurus-test /bin/bash -c "apt-get -qq update"
+  - docker exec taurus-test /bin/bash -c "apt-get install -y python-sphinx-rtd-theme"
```

Comment:
I would do this (install sphinx-rtd-image) in the docker images instead.

Reply:
I agree, but I don't know if I have access to modify the taurus-test docker files. @cpascual, could you add the python-sphinx-rtd-theme to the dockers?

Reply:
I just added it to debian-stretch and debian-buster images.
I am not sure we should add it to debian-jessie